admin_guide/enabling_EFK.adoc

```diff
@@ -0,0 +1,357 @@
+"v1.1" creates "openshift/origin-logging-deployer:v1.1".
+
+|`*ES_INSTANCE_RAM*`
+|Amount of RAM to reserve per ElasticSearch instance. The default is 8GB, and it must be at least 512MB.
+
+|`*ES_CLUSTER_SIZE*`
+|The amount of instances of ElasticSearch to deploy. Redundancy requires at
+least three, and more can be used for scaling.
+
+|`*ENABLE_OPS_CLUSTER*`
+|If set to "true", configures a second ElasticSearch cluster and Kibana for
+operations logs.
+
+|`*KIBANA_OPS_HOSTNAME*`, `*ES_OPS_INSTANCE_RAM*`, `*ES_OPS_CLUSTER_SIZE*`
+|Variables for the operations log cluster.
+|===
++
+When setting the `*ENABLE_OPS_CLUSTER*` parameter to 'true', Fluentd splits logs
+between the ElasticSearch cluster and a cluster reserved for operations logs.
+This means a second ElasticSearch and Kibana are deployed. The deployments are
+distinguishable by the '-op' included in their names.
+
+. Next, as a cluster administrator, deploy the templates that were created by
+the deployer:
++
+====
+----
+$ oc process logging-support-template | oc create -f -
+----
+====
+
+. Enable the Fluentd service account by editing the security context:
++
+====
+----
+$ oc edit scc/privileged
+----
+====
++
+And add the following user to the file:
++
+====
+----
+- system:serviceaccount:logging:aggregated-logging-fluentd
+----
+====
+
+. Give the service account permission to read labels from all pods:
++
+====
+----
+$ openshift admin policy add-cluster-role-to-user cluster-reader system:serviceaccount:logging:aggregated-logging-fluentd
+----
+====
+
+== Post-deployment Configuration
+
+=== ElasticSearch
+
+All pods in OpenShift deployments can safely share any volumes specified in the
+deployment. With ElasticSearch, all pods must have their own storage.
+
+The ability to specify multiple volumes to be allocated to each instance in a
+deployment is currently being completed. Currently, multiple deployments are
+used in order to scale ElasticSearch. To view all current deployments used by
+ElasticSearch:
+
+====
+----
+$ oc get dc --selector logging-infra=elasticsearch
+----
+====
+
+To scale ElasticSearch deployments, create and add more deployments, being aware
+of the cluster parameter restrictions. The deployer uses a template to create
+ElasticSearch deployments. These deployments will be named differently, but will
+all have the 'service/logging-es-cluster' prefix:
+
+====
+----
+$ oc process logging-es-template | oc create -f -
+----
+====
+
+*ElasticSearch Storage*
+
+The deployer creates a ephemeral deployment in which all of a pod's data is lost
+upon restart. For production,
+link:../admin_guide/persistent_storage_nfs.html[specify a persistent storage
+volume] for each ElasticSearch deployment. You can use the 'oc volume' command
+to add a volume to a deployment:
+
+====
+----
+$ oc volume dc/logging-es-rca2m9u8 \
+          --add --overwrite --name=elasticsearch-storage \
+          --type=hostPath --path=/path/to/storage
```

Comment:
I don't think we do have other examples... but I did neaten this up as suggested.